diff --git a/config.def.h b/config.def.h
index a16f284..8b57251 100644
--- a/config.def.h
+++ b/config.def.h
@@ -4,7 +4,7 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-  "monospace:size=10"
+  "monospace:size=10", "Noto Color Emoji:size=11"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
