diff --git a/config.def.h b/config.def.h
index 9e94f31..199a121 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,8 +5,9 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "mono:pixelsize=15:antialias=true:autohint=true";
+static char *font2[] = { "JoyPixels:pixelsize=10:antialias=true:autohint=true" };
+static int borderpx = 15;
 
 /*
  * What program is execed by st depends of these precedence rules:
@@ -96,30 +97,32 @@ unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
   /* 8 normal colors */
-  "black",
-  "red3",
-  "green3",
-  "yellow3",
-  "blue2",
-  "magenta3",
-  "cyan3",
-  "gray90",
+  "#1d2021", /* black   */
+  "#cc241d", /* red     */
+  "#98971a", /* green   */
+  "#d79921", /* yellow  */
+  "#458588", /* blue    */
+  "#b16286", /* magenta */
+  "#689d6a", /* cyan    */
+  "#a89984", /* white   */
 
   /* 8 bright colors */
-  "gray50",
-  "red",
-  "green",
-  "yellow",
-  "#5c5cff",
-  "magenta",
-  "cyan",
-  "white",
+  "#928374", /* black   */
+  "#fb4934", /* red     */
+  "#b8bb26", /* green   */
+  "#fabd2f", /* yellow  */
+  "#83a598", /* blue    */
+  "#d3869b", /* magenta */
+  "#8ec07c", /* cyan    */
+  "#ebdbb2", /* white   */
 
   [255] = 0,
 
   /* more colors can be added after 255 to use with DefaultXX */
-  "#cccccc",
-  "#555555",
+  "#add8e6", /* 256 -> cursor */
+  "#555555", /* 257 -> rev cursor*/
+  "#1d2021", /* background */
+  "#ebdbb2", /* foreground */
 };
 
 
@@ -127,8 +130,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
+unsigned int defaultfg = 259;
+unsigned int defaultbg = 258;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
@@ -174,6 +177,8 @@ static uint forcemousemod = ShiftMask;
  */
 static MouseShortcut mshortcuts[] = {
   /* mask                 button   function        argument       release */
+  { ShiftMask,            Button4, kscrollup,      {.i = 1} },
+  { ShiftMask,            Button5, kscrolldown,    {.i = 1} },
   { XK_ANY_MOD,           Button2, selpaste,       {.i = 0},      1 },
   { ShiftMask,            Button4, ttysend,        {.s = "\033[5;2~"} },
   { XK_ANY_MOD,           Button4, ttysend,        {.s = "\031"} },
@@ -183,7 +188,7 @@ static MouseShortcut mshortcuts[] = {
 
 /* Internal keyboard shortcuts. */
 #define MODKEY Mod1Mask
-#define TERMMOD (ControlMask|ShiftMask)
+#define TERMMOD (Mod1Mask|ShiftMask)
 
 static Shortcut shortcuts[] = {
   /* mask                 keysym          function        argument */
@@ -193,12 +198,20 @@ static Shortcut shortcuts[] = {
   { XK_ANY_MOD,           XK_Print,       printsel,       {.i =  0} },
   { TERMMOD,              XK_Prior,       zoom,           {.f = +1} },
   { TERMMOD,              XK_Next,        zoom,           {.f = -1} },
-  { TERMMOD,              XK_Home,        zoomreset,      {.f =  0} },
-  { TERMMOD,              XK_C,           clipcopy,       {.i =  0} },
-  { TERMMOD,              XK_V,           clippaste,      {.i =  0} },
-  { TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
-  { ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
+  { MODKEY,               XK_Home,        zoomreset,      {.f =  0} },
+  { MODKEY,               XK_c,           clipcopy,       {.i =  0} },
+  { MODKEY,               XK_v,           clippaste,      {.i =  0} },
+  { MODKEY,               XK_p,           selpaste,       {.i =  0} },
+  { ShiftMask,            XK_Insert,      clippaste,      {.i =  0} },
   { TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
+  { ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },
+  { ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} },
+  { MODKEY,               XK_Page_Up,     kscrollup,      {.i = -1} },
+  { MODKEY,               XK_Page_Down,   kscrolldown,    {.i = -1} },
+  { MODKEY,               XK_k,           kscrollup,      {.i =  1} },
+  { MODKEY,               XK_j,           kscrolldown,    {.i =  1} },
+  { TERMMOD,              XK_K,           zoom,           {.f = +1} },
+  { TERMMOD,              XK_J,           zoom,           {.f = -1} },
 };
 
 /*
